daemon/models.go

```diff
@@ -12,4 +12,5 @@ type HandshakeResponse struct {
 type RequestExecution struct {
 	RequestResponses []request.ExecutedRequestResponse
 	ErrorMessage     string
+	PostProcessError string
```

Comment:
Another naming nit: it's slightly odd to see `ErrorMessage` but then not see `PostProcessErrorMessage`?

Reply:
Agree, I initially had it that way but it was too long so I changed it.
I know I'm kind of repeating myself but this will also change. For the result this will have to become a struct which would include normal output, error messages and such.
Right now, all output goes to the daemon log which is useless.